blockchains/cosmos/validators/list.json

```diff
@@ -310,5 +310,11 @@
         "name": "Genesis Lab",
         "description": "Genesis Lab is a validation nodes operator in PoS networks and blockchain-focused software development company",
         "website": "https://genesislab.net"
+    },
```

Comment:
`+495 KB` - make file smaller and 256 by 256 png

Reply:
there are bigger files in the repo

Reply:
done